src/rviz/visualization_frame.h

```diff
@@ -32,6 +32,7 @@
 
 #include <boost/shared_ptr.hpp>
 
+#include <QCheckBox>
```

Comment:
is this change just leftover from https://github.com/ros-visualization/rviz/pull/1238?